src/app.rs

```diff
@@ -528,13 +412,14 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
             &self.cfg.document,
             &self.mailbox(),
             &self.clone(),
-            &self.cfg.mount_point,
+            self.cfg.mount_point(),
```

Comment:
Not sure that you can remove the `mount_point` in this case. It's used to place the newly created DOM elements, so you'll always need it around to update the DOM after updating the VDOM.